app/controllers/entries_controller.rb

```diff
@@ -3,11 +3,14 @@ def day_popup_form
     date = Date.parse(params[:date])
     day = current_user.days.find_by(date: date)
     @entry = day.find_short_entry
+    @memories = day.be_real_memories
   end
 
   def update
     @entry = Entry.find_by(id: params[:id])
     @entry.update(entry_params)
+    @memories = @entry.user.days.find_by(date: @entry.date).be_real_memories
```

Comment:
Shouldn't it just be this? 
```suggestion
    @memories = @entry.day.be_real_memories
```

Reply:
Yep, was confused at that connection.